src/main/groovy/org/exoplatform/platform/am/cli/CommandLineParameters.groovy

```diff
@@ -46,18 +46,20 @@ class CommandLineParameters {
   static final String OFFLINE_LONG_OPT = "--offline"
   static final String CONFLICT_LONG_OPT = "--conflict"
   static final String FORCE_LONG_OPT = "--force"
+  static final String CRCCHECK_OPT = "--crc"
 
   // CLI Commands
   static final String LIST_CMD = "list"
   static final String DESCRIBE_CMD = "describe"
   static final String INSTALL_CMD = "install"
   static final String UNINSTALL_CMD = "uninstall"
+  static final String OFFINSTALL_CMD = "apply"
```

Comment:
Why not using APPLY_CMD as a variable name ?

Reply:
OK Changes Applied